deploy/kubernetes/postee.yaml

```diff
@@ -267,7 +267,7 @@ spec:
             - name: POSTEE_UI_PORT
               value: "8000"
             - name: POSTEE_UI_UPDATE_URL
-              value: "http://postee-svc:8080"
+              value: "http://postee-svc:8082"
             - name: POSTEE_ADMIN_USER
               value: admin
             - name: POSTEE_ADMIN_PASSWORD
```

Comment:
@krol3 can we take this from the values file? Default password is not a best practice.

Reply:
@jerbia I removed that part, and improve the readme about this. 

To edit the default Postee-UI user

```
kubectl -n postee set env deployment/my-posteeui -e POSTEE_ADMIN_USER=testabc -e POSTEE_ADMIN_PASSWORD=password
```